orttraining/orttraining/core/optimizer/graph_transformer_utils.cc

```diff
@@ -90,6 +90,7 @@ std::vector<std::unique_ptr<GraphTransformer>> GeneratePreTrainingTransformers(
       }
 
       transformers.emplace_back(onnxruntime::make_unique<ConstantFolding>(execution_provider, compatible_eps, weights_to_train));
+      transformers.emplace_back(onnxruntime::make_unique<ReshapeFusion>(compatible_eps));
```

Comment:
nit... maybe there is no difference, since we apply multiple passes of optimizations. 
But should ReshapeFusion goes before ConstantFolding, since ConstantFolding depends a lot on shape.

Reply:
reshape fusion also depends on the constant folding (like fuse the unqueeze input to constant), so they have to run multiple iterations. which one in front doesn't matter, i think.